### DISCUSSION TITLE
Why I don't have the window top bar?

### DISCUSSION THREAD

**Discussion by anonimo82** [nathaneltitane/dextop#23] [General]
Hello everybody.
I gave Dextop another try, and I noticed I don't have the top window bar in my windows in XFCE.
Why? I guess something went wrong with my installation, is it that?

Here is a screenshot:
![Screenshot_20230503-174312_bVNC Free](https://user-images.githubusercontent.com/38261009/235967943-e78ced1e-acd2-4bbe-a11a-2374dc5d6391.jpg)

Notice I don't have the x button and so.

**Comment by nathaneltitane** [comment]
Could be an issue with the themes download: Dextop downloads and applies Arc Theme as default.
You can use the xfce settings to change it back to whatever default.

PLease let me know if that worked - I will also investigate why this did not work in your case.

**Comment by nathaneltitane** [comment]
> Hello everybody. I gave Dextop another try, and I noticed I don't have the top window bar in my windows in XFCE. Why? I guess something went wrong with my installation, is it that?
> 
> Here is a screenshot: ![Screenshot_20230503-174312_bVNC Free](https://user-images.githubusercontent.com/38261009/235967943-e78ced1e-acd2-4bbe-a11a-2374dc5d6391.jpg)
> 
> Notice I don't have the x button and so.

Have you tried reinstalling (or installing, if it didn't) Arc Theme ?

**Comment by anonimo82** [comment]
No, how can I do? I guess with "pkg i packageName", but how is the Arc Theme package named?

**Reply by nathaneltitane** [reply]
> No, how can I do? I guess with "pkg i packageName", but how is the Arc Theme package named?

Both setups use apt and dpkg - are you familiar with Linux-Based OSes at all?

pkg is a wrapper for dpkg and apt/apt-get

You can search for Arc theme using apt-get-search arc-theme on either Termux or the Ubuntu container.

Please do your homework...

**Comment by anonimo82** [comment]
arc-theme is not listed in my search results.
I guess that's the problem 

Nevermind, I have it in the container and it's installed

**Reply by nathaneltitane** [reply]
Fixed. Termux repositories removed the package. Now copied from container on default install.
